src/dialog/provider.js

```diff
@@ -127,31 +92,17 @@ export default class FlowProvider extends EventEmitter2 {
 
     // What goes in the ui.json file
     const uiContent = {
-      nodes: flow.nodes.map(node => ({
-        id: node.id,
-        position: { x: node.x, y: node.y }
-      })),
+      nodes: flow.nodes.map(node => ({ id: node.id, position: { x: node.x, y: node.y } })),
```

Comment:
NIT but can use `_.pick(node, 'x', 'y')`, in line with the other places in this file

Reply:
OK, changed that.